src/payment/checkout/payment-form/PaymentForm.messages.jsx

```diff
@@ -26,6 +26,11 @@ const messages = defineMessages({
     defaultMessage: 'This field is required',
     description: 'The form field feedback text for missing required field error.',
   },
+  'payment.form.errors.ascii.name': {
+    id: 'payment.form.errors.ascii.name',
+    defaultMessage: 'We apologize for the inconvenience but for the time being we require ASCII characters in the name field. We are working on addressing this and appreciate your patience.',
```

Comment:
Notes:
* Our users may not understand what ASCII characters mean in practice. It may be helpful to provide a hint on what characters are allowed [e.g. “we require names only have unaccented upper or lower case letters of the Latin alphabet (A–Z or a-z)”]
* If I’m reading the regex and code correctly, there may be characters in the ASCII character set (like @) that we do not accept. This may be confusing for users because the accepted characters in practice would be different from what the error message says we accept.

Reply:
This is the wording I got from Seth, I'm inclined to just run with it; this is just temporary until we get back and fix the problem.